### ISSUE TITLE
sync validator errors are suppressed when using an async validator

### ISSUE THREAD

**Issue by giniedp** [angular/angular#15389]
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
When using async validators together with sync validators, the errors from sync validators have no effect, when async validator is resolved

**Expected behavior**
Even when an async validator resolves without errors, the validation errors of the sync validators (e.g. required) should be reported

**Minimal reproduction of the problem with instructions**
here is the plunker https://plnkr.co/edit/JOVCMycssT0c7vcZa6jZ

initially the input field is marked as invalid due to the `required` validator. When input is changed, the custom async validator kicks in. When the input is cleared, the async validator reports a success and this marks the field as valid.

**Comment by DzmitryShylovich** [comment]
Duplicate of https://github.com/angular/angular/issues/15235

**Comment by kara** [comment]
Yep, this was fixed recently and should be out in 4.0.0-beta.6 versions and later.

**Comment by giniedp** [comment]
thank you. upgraded to angular 4. works as expected.
